src/main/java/org/thoughtcrime/securesms/qr/QrScanFragment.java

```diff
@@ -106,7 +106,7 @@ public MyCaptureManager(Activity activity, DecoratedBarcodeView barcodeView) {
         // which makes _showing_ the QR-code impossible if scanning goes wrong.
         // therefore, we only show a non-disturbing error here.
         @Override
-        protected void displayFrameworkBugMessageAndExit() {
+        protected void displayFrameworkBugMessageAndExit(String message) {
             Toast.makeText(myActivity, R.string.zxing_msg_camera_framework_bug, Toast.LENGTH_SHORT).show();
```

Comment:
it probably makes sense to make use of the given `message`. looking at the source code, i found eg. [this](https://github.com/journeyapps/zxing-android-embedded/blob/master/zxing-android-embedded/src/com/journeyapps/barcodescanner/CaptureManager.java#L406) which does the following:

```suggestion
            if (message.isEmpty()) {
                message = activity.getString(R.string.zxing_msg_camera_framework_bug);
            }
            Toast.makeText(myActivity, message, Toast.LENGTH_SHORT).show();
```

Reply:
just to be safe I will check if message is not null